bftengine/tests/simpleTest/replica.cpp

```diff
@@ -378,7 +381,15 @@ int main(int argc, char **argv) {
 
   simpleAppState.st = st;
 
-  replica = Replica::createNewReplica(&replicaConfig, &simpleAppState, st, comm, nullptr);
+  replicaConfig.statusReportTimerMillisec = 3000;
+  replicaConfig.viewChangeTimerMillisec = 20000;
+  ostringstream dbFile;
+  dbFile << "metadataStorageTest_" << replicaConfig.replicaId << ".txt";
+  remove(dbFile.str().c_str());
+  MetadataStorage *fileStorage = new FileStorage(replicaLogger, dbFile.str());
+
+  replica = Replica::createNewReplica(&replicaConfig, &simpleAppState, st, comm, fileStorage);
+  //replica = Replica::createNewReplica(&replicaConfig, &simpleAppState, st, comm, nullptr);
```

Comment:
Let's delete this line if we aren't using it.

Reply:
Igor is improving this code now. Don't like to touch it to minimize merging issues.